src/page-modules/contact/ticketing/forms/app/appTicketingForm.tsx

```diff
@@ -0,0 +1,148 @@
+import { PageText, useTranslation } from '@atb/translations';
+import { ticketingFormEvents } from '../../events';
+import { TicketingContextType } from '../../ticketingStateMachine';
+import { Typo } from '@atb/components/typography';
+import { SectionCard, Input, Textarea, FileInput } from '../../../components';
+
+type AppTicketingFormProps = {
+  state: { context: TicketingContextType };
+  send: (event: typeof ticketingFormEvents) => void;
+};
+
+export const AppTicketingForm = ({ state, send }: AppTicketingFormProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      <SectionCard title={t(PageText.Contact.ticketing.app.appTicketing.title)}>
+        <Typo.p textType="body__primary">
+          {t(PageText.Contact.input.orderId.info)}
+        </Typo.p>
+        <Input
+          label={PageText.Contact.input.orderId.label}
+          type="text"
+          name="orderId"
+          description={t(PageText.Contact.input.orderId.description)}
+          value={state.context.orderId || ''}
+          errorMessage={state.context?.errorMessages['orderId']?.[0]}
+          onChange={(e) =>
+            send({
+              type: 'ON_INPUT_CHANGE',
+              inputName: 'orderId',
+              value: e.target.value,
+            })
+          }
+        />
+      </SectionCard>
+      <SectionCard title={t(PageText.Contact.input.question.title)}>
+        <Typo.p textType="body__primary">
+          {t(PageText.Contact.input.question.info)}
+        </Typo.p>
+        <Textarea
+          value={state.context.question || ''}
+          onChange={(e) =>
+            send({
+              type: 'ON_INPUT_CHANGE',
+              inputName: 'question',
+              value: e.target.value,
+            })
+          }
+          error={
+            state.context.errorMessages['question']?.[0]
+              ? t(state.context.errorMessages['question']?.[0])
```

Comment:
Could remove the optional chaining here, since we know that it exists based on the check on the line above. But not a big deal.